src/components/CommentsCard.jsx

```diff
@@ -1,7 +1,42 @@
+import { fetchCommentsByArticle } from "../../utils/api";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
 export default function CommentsCard() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState();
+  const params = useParams();
```

Comment:
What's this variable doing for you, Gabriel? I can't see it being used anywhere in the code other than on a console log on Line 18?